velox/exec/prefixsort/PrefixSortEncoder.h

```diff
@@ -46,10 +51,18 @@ class PrefixSortEncoder {
   FOLLY_ALWAYS_INLINE void encode(std::optional<T> value, char* dest) const {
     if (value.has_value()) {
       dest[0] = nullsFirst_ ? 1 : 0;
-      encodeNoNulls(value.value(), dest + 1);
```

Comment:
I don’t prefer to add an if statement to the hot code. Can we use encodeNoNulls for compatibility processing? If not, can you try refactoring this method?